_api_app/app/Sites/Sections/SiteSectionsDataService.php

```diff
@@ -308,48 +308,59 @@ public function saveValueByPath($path, $value)
             $old_name = $sections['section'][$order]['name'];
             $old_title = isset($sections['section'][$order]['title']) ? $sections['section'][$order]['title'] : '';
             $new_name = $this->getUniqueSlug($old_name, $value);
+            $success=false;
```

Comment:
Use spaces around assignment operator - `$success=false` -> `$success = false`;
Try to follow the code style you see around.